spec/models/terms_of_service_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe TermsOfService do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
My preference would be to take the first suggestion from this line, but either of them is better than neither of them.